### ISSUE TITLE
Why gradient clipping is removed?

### ISSUE THREAD

**Issue by chesschi** [Jongchan/tensorflow-vdsr#28]
1. Why all the source code related to gradient clipping in VDSR.py in [8b59484fea473d35acfa6f0788739e5888dc437a ](https://github.com/Jongchan/tensorflow-vdsr/blob/8b59484fea473d35acfa6f0788739e5888dc437a/VDSR.py#L122) is removed from master branch? Is it not working?

https://github.com/Jongchan/tensorflow-vdsr/blob/8b59484fea473d35acfa6f0788739e5888dc437a/VDSR.py#L122

2. In addition, I have seen that you are trying to divide learning rate by 10 for each iteration when `USE_QUEUE_LOADING` is enabled, i.e.
https://github.com/Jongchan/tensorflow-vdsr/blob/8b59484fea473d35acfa6f0788739e5888dc437a/VDSR.py#L242
but do not re-calculate the norm for gradient clipping when `USE_QUEUE_LOADING` is disabled. Please can you kindly explain why is the case?

Many thanks!

**Comment by Jongchan** [comment]
I removed the gradient clipping because Adam optimizer can handle the gradient scale by itself.
Please refer to the comments in [this issue](https://github.com/Jongchan/tensorflow-vdsr/issues/5)

About the norm, I think it is calculated according to the current learning rate, as written in
https://github.com/Jongchan/tensorflow-vdsr/blob/8b59484fea473d35acfa6f0788739e5888dc437a/VDSR.py#L212

If there are some mistakes, please point it out. Thanks!

**Comment by chesschi** [comment]
Thank you for your reply.

If you had a look in the source code VDSR.py for the else-statement that `USE_QUEUE_LOADING` is not enabled, you can see that norm is not re-calculated and no gradient clipping is applied. I am curious why you did not do it when `USE_QUEUE_LOADING` is not enabled.

https://github.com/Jongchan/tensorflow-vdsr/blob/8b59484fea473d35acfa6f0788739e5888dc437a/VDSR.py#L243

**Comment by Jongchan** [comment]
ah, it is not intentional. Because queue loading is like 2~5 times faster, I always use it as a default option. I did not care about the rest of the code.

Anyways, there are some mistakes in my old codes, so if you are planning to use it, please be careful. If you have any question or unexpected results, please write an issue, or directly email me (which will be faster...).

Sorry for any confusion made.
